### PR TITLE
Release lacework agent v6.12.0 charts

### PR DESCRIPTION
**Pull request by catz-lw**
Please note there are two commits. The first is the verbatim set of v6.12.0 charts from the original v6.12.0 release, and the second newer commit includes fixes for https://github.com/lacework/helm-charts/issues/224

### CODE REVIEW

**Review by nschmeller — APPROVED** [review]
Are you planning to squash or rebase these commits?

File: index.yaml [hunk 1]
```diff
@@ -900,7 +921,7 @@ entries:
     version: 6.11.0
   - apiVersion: v2
     appVersion: "1.0"
-    created: "2024-01-09T17:18:19.336479925Z"
+    created: "2024-02-21T14:04:56.154085-08:00"
```
Comment (catz-lw) [inline index.yaml]: This looks wrong. The digest for 6.11 should not change.

Comment (catz-lw) [reply index.yaml]: I had actually changed the digest here, too. I left the comment on a line that doesn't show the context.

Comment (catz-lw) [reply index.yaml]: https://github.com/lacework/helm-charts/commit/4e3ecaaa94fbdf80d5790ece97c96954b2b090c4#r139131323

**Pull request merged**